### PR TITLE
arpwatch: Import from the packages archive

### PR DESCRIPTION
**Pull request by malaiwah**
Maintainer: Michel Belleau / @malaiwah
Compile tested: Yes, Atheros AR7xxx/AR9xxx (GL.iNet GL-AR150)
Run tested: Yes, OpenWRT-snapshot (2018-12-28)

Description:
arpwatch is a tool for monitoring Address Resolution Protocol traffic on a computer network. It generates a log of observed pairing of IP addresses with MAC addresses along with a timestamp when the pairing appeared on the network. It also has the option of sending an email to an administrator when a pairing changes or is added.

### CODE REVIEW

**Comment by cshoredaniel** [comment]
IIRC it's BSD-4-Clause not BSD-3-Clause -- did they change the license?

**Comment by malaiwah** [comment]
I admit I blindly copy/pasted from the Wikipedia article.
I will a a deeper look from the source.

https://en.wikipedia.org/wiki/Arpwatch

**Comment by diizzyy** [comment]
@malaiwah 
If you're deadset on arpwatch you're probably better off importing Alpinelinux's patches and using autotools but I still think you should look for something that's still in active development.

**Comment by malaiwah** [comment]
I am deadset, arpwatch is plain simple and even dead still is useful. addrwatch is similar but not as lightweight -- it records all ARP events that are happenning and I just want to know the changes, which arpwatch does without sweat. I'll look at the comments and the Alpinelinux patches.

**Comment by malaiwah** [comment]
> I admit I blindly copy/pasted from the Wikipedia article. I will a a deeper look from the source. 

I will go with your comment and change this to 4-Clause (Original) -- sources aren't clear and I believe the 4-Clause was the one used at that time.

commit 75e2abc0b9edbf8cace6cd109e888085129cbfa6

File: net/arpwatch/Makefile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+PKG_NAME:=arpwatch
+PKG_VERSION:=2.1a15
+PKG_RELEASE:=2
+
+PKG_MAINTAINER:=Michel Belleau <michel.belleau@malaiwah.com>
+PKG_LICENSE:=BSD-3-Clause
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=ftp://ftp.free.fr/.mirrors1/ftp.gentoo.org/distfiles/
+PKG_MD5SUM:=cebfeb99c4a7c2a6cee2564770415fe7
```
[the remark this answers is not in the document]
Comment (malaiwah) [reply net/arpwatch/Makefile]: commit 380b84702befab4a14ccfc87c546694fc85ba517

**Comment by diizzyy** [comment]
Import all patches and use autotools instead
arp.dat is also not a configuration file as far as I can tell and shouldn't be in /etc at all (having a quick look at the documentation)

**Comment by diizzyy** [comment]
@malaiwah 
Ping?

**Comment by neheb** [comment]
ping @malaiwah

**Comment by malaiwah** [comment]
Time goes by so fast, but getting that PR dressed up is still on my todo.

**Comment by erdoukki** [comment]
> Compared to https://git.alpinelinux.org/cgit/aports/tree/main/arpwatch?h=master this PR needs work (and lacks a lot of patches)
> I would perfer that we import https://github.com/fln/addrwatch instead which isn't dead upstream and seems to do pretty much the same thing.

addrwatch do not send email notifications !

**Comment by diizzyy** [comment]
Closing this due to timeout

**Pull request closed without merging**